Makefile

```diff
@@ -26,6 +27,9 @@ help:
 install:
 	pip install --no-deps -e .
 
+package:
+	python setup.py sdist bdist_wheel
```

Comment:
Just wondering, when do we want to use this `make package` command? And what is the different between `python setup.py sdist bdist_wheel` and `python setup.py sdist --formats=zip` used in 'ci_tests.yaml'?

Reply:
> Just wondering, when do we want to use this `make package` command? 

I was trying to address issue #904, so had to run the commands many times. `make pacakge` is easier to type.

> And what is the different between `python setup.py sdist bdist_wheel` and `python setup.py sdist --formats=zip` used in 'ci_tests.yaml'?

`sdist` is the source distribution, and `bdist_wheel` is the binary distribution (in wheel format). They are uploaded to PyPI (https://pypi.org/project/pygmt/#files). 

In `ci_tests.yaml`, we need to install PyGMT, so we only need the source distribution (`sdist`).

Reply:
> > Just wondering, when do we want to use this `make package` command?
> 
> I was trying to address issue #904, so had to run the commands many times. `make pacakge` is easier to type.

Ok, fair enough. I would just use the [CTRL+R reverse-i-search](https://www.chriswrites.com/search-terminal-history/) command to save on typing an old command but `make package` works too.

> > And what is the different between `python setup.py sdist bdist_wheel` and `python setup.py sdist --formats=zip` used in 'ci_tests.yaml'?
> 
> `sdist` is the source distribution, and `bdist_wheel` is the binary distribution (in wheel format). They are uploaded to PyPI (https://pypi.org/project/pygmt/#files).
> 
> In `ci_tests.yaml`, we need to install PyGMT, so we only need the source distribution (`sdist`).

Got it, thanks for the explanation.